packages/slate/src/changes/with-schema.js

```diff
@@ -121,12 +104,22 @@ function normalizeNodeAndChildren(change, node, schema) {
  * @return {Node}
  */
 
-function refindNode(change, node) {
+function refindNode(change, node, lastPath) {
   const { value } = change
   const { document } = value
-  return node.object == 'document'
-    ? document
-    : document.getDescendant(node.key)
+  if (node.object === 'document') {
```

Comment:
Can you add breaks about these `if` statements to match the rest of the code?

Reply:
Yes, I will